gpu/mini-apps/affinity/main.cpp

```diff
@@ -0,0 +1,110 @@
+//Chris will add more comments 
+//Setup affinity of GPUs with cpus. 
```

Comment:
Maybe this is obvious but can someone just explain to me what the word "affinity" means in this context?